packages/manager/modules/veeam-enterprise/src/translations/Messages_fr_FR.json

```
@@ -1,4 +1,6 @@
 {
   "veeam_enterprise_title": "Veeam Enterprise",
-  "veeam_enterprise_order": "Commander"
+  "veeam_enterprise_order": "Commander",
+  "veeam_enterprise_list_service_name": "Nom",
+  "veeam_enterprise_list_activation_status": "Status"
```

Not exist in common translation ?

---

It exists https://github.com/ovh/manager/blob/master/packages/manager/modules/common-translations/public/translations/status/Messages_fr_FR.json.
```suggestion
  "veeam_enterprise_order": "Commander",
  "veeam_enterprise_list_service_name": "Nom",
  "veeam_enterprise_list_activation_status": "Statut"
```